### PR TITLE
Interactivity and Pop-ups Guide

### PR DESCRIPTION
**Pull request by borja-munoz**

### CODE REVIEW

File: docs/guides/03-interactivity-popups.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+        src="/developers/web-sdk/examples/maps/guides/interactivity-popups/step-1.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+> View this step [here](/developers/web-sdk/examples/maps/guides/interactivity-popups/step-1.html)
+
+We can specify custom highlighting properties directly use the deck.gl properties like `getFillColor` or `getLineColor`.
```
Comment (VictorVelarde) [inline docs/guides/03-interactivity-popups.md]: "directly using" ?

File: docs/guides/03-interactivity-popups.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
+> You can explore the final step [here](/developers/web-sdk/examples/maps/guides/interactivity-popups/step-7.html)
+
+```html
+<!DOCTYPE html>
```
Comment (VictorVelarde) [inline docs/guides/03-interactivity-popups.md:226]: duplicated DOCTYPE

File: docs/guides/03-interactivity-popups.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+
+    <!-- Include Mapbox GL from the Mabpox CDN -->
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js'></script>
+
+    <!-- Include Web SDK from the CARTO CDN -->
+    <script src="https://libs.cartocdn.com/web-sdk/v1.0.0-alpha/index.min.js"></script>
+
+    <!-- Include Airship from the CARTO CDN -->
+    <script type="module" src="https://libs.cartocdn.com/airship-components/v2.4.0/airship/airship.esm.js"></script>
+    <script nomodule="" src="https://libs.cartocdn.com/airship-components/v2.4.0/airship/airship.js"></script>
```
Comment (VictorVelarde) [inline docs/guides/03-interactivity-popups.md:267]: Nice addition! We should add that in other airship uses in our examples, as chrome suggests

**Review by VictorVelarde — APPROVED** [review]
Also another really nice guide, great job!

File: docs/guides/03-interactivity-popups.md (hunk 1 above)
Comment (borja-munoz) [reply docs/guides/03-interactivity-popups.md]: Fixed

File: docs/guides/03-interactivity-popups.md (hunk 2 above)
Comment (borja-munoz) [reply docs/guides/03-interactivity-popups.md:226]: Fixed

**Pull request merged**